js/app/cookies-banner.js

```diff
@@ -1,24 +1,39 @@
+// get preferences cookie
+var cookiesSet = hasCookiesPreferencesSet()
+
 // cookies settings
 var oneYearInSeconds = 31622400;
 var url = window.location.hostname;
 var cookiesDomain = extractDomainFromUrl(url);
 var cookiesPreference = true;
 var encodedCookiesPolicy = "%7B%22essential%22%3Atrue%2C%22usage%22%3Atrue%7D";
 var cookiesPath = "/";
+var cookiesBanner = $(".js-cookies-banner-form");
+
+document.addEventListener("DOMContentLoaded", determineWhetherToRenderBanner())
+
+function determineWhetherToRenderBanner() {
+    cookiesBanner.addClass("hidden");
```

Comment:
I would include a comment here - I can see someone in the future editing this function then being like, "Ooo I can see an optimisation here, by only hiding if the cookie pref has been set". I think it would be worth a mention that this is done to prevent elements moving around during load.

Reply:
I think the comment won't be needed anymore bearing in mind the updated code you've suggested at the bottom